.github/workflows/release-periodically.yml

```diff
@@ -45,21 +45,20 @@ jobs:
           # 128 -> fatal: no tag exactly matches 'c88a1272ffbc002042fd62ed409734c1c150cc5f'
           # 0 -> anaconda-38.6-1
           if git describe --tags --exact-match ; then
-            echo "Already tagged at HEAD, will not push another tag."
-            echo "::set-output name=can-tag::false"
+            echo "::warning:: Already tagged at HEAD, will not push a release commit."
           else
-            echo "::set-output name=can-tag::true"
+            echo "can_bump_version=true" >> $GITHUB_OUTPUT
```

Comment:
Interesting, I wasn't aware of this solution.

Reply:
The solution with `set-output` is deprecated: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Good to know. Thanks!